### PR TITLE
Add blog guidelines to Style Guide

### PR DESCRIPTION
**Pull request by cyrichardson**

### CODE REVIEW

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/build-87c34df535/docs/style-guide/). :bow:

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/build-4fa2ae0bec/docs/style-guide/)

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/build-5a15fcf2f6/docs/style-guide/).

File: style-guide/blog-guidelines.rst [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+------------------------
+
+When you write your content, we recommend the following rules of thumb:
+
+-  Length: Rackspace blog posts generally run between 500-1,000 words, but let
+   the content be your guide. Take the space you need to tell your story, but
+   don’t pad it unnecessarily. Going long is okay, too, as long as it’s worth
+   reading.
+
+-  Tools: We recommend using a word processor, like MicroSoft Word, or a text
```
Comment (stephamon) [inline style-guide/blog-guidelines.rst]: I don't think "Microsoft" is supposed to have a capital "S" in it

File: style-guide/blog-guidelines.rst [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+-  Avoid jargon! Rackers (and the entire tech industry) tend to speak in
+   jargon. Unless your post is aimed at a technical audience, use plain
+   English. There is never a reason to use business jargon.
+
+-  Link link link: A first product mention? Link to the product page. Outside
+   accolades? Link to it. Citing a study or source or article? Link to it.
+   Have we written about it before? Link to it. Mention a partner or customer?
+   Link to their site.
+
+-  Keep it concise: The internet have shrunken everyone’s attention span.
```
Comment (stephamon) [inline style-guide/blog-guidelines.rst]: The Internet has shrunken...

File: style-guide/blog-guidelines.rst [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+     information
+
+Be consistent in style with other blogs on the Rackspace Developer Blog site.
+
+Bullet points are useful for listing features.
+
+Think “be helpful” rather than trying to sell a product or service. WHAT does
+the product do? HOW does it help the customer?
+
+-  Write a good “lede”: When crafting your lede, consider how the information
```
Comment (stephamon) [inline style-guide/blog-guidelines.rst]: Should "lede" be in quotes here for a second time? (see line 66)

File: style-guide/blog-guidelines.rst [hunk 4]
```diff
@@ -0,0 +1,181 @@
...
+   the content be your guide. Take the space you need to tell your story, but
+   don’t pad it unnecessarily. Going long is okay, too, as long as it’s worth
+   reading.
+
+-  Tools: We recommend using a word processor, like MicroSoft Word, or a text
+   editor for your writing.
+
+-  General format:
+
+   - An attention-grabbing opening, known in the news biz as a “lede”
```
Comment (stephamon) [inline style-guide/blog-guidelines.rst]: Given that we recommend avoiding jargon further down, should we expand "biz" to "business"?

File: style-guide/blog-guidelines.rst (hunk 1 above)
Comment (cyrichardson) [reply style-guide/blog-guidelines.rst]: fixed

File: style-guide/blog-guidelines.rst (hunk 2 above)
Comment (cyrichardson) [reply style-guide/blog-guidelines.rst]: fixed

File: style-guide/blog-guidelines.rst (hunk 3 above)
Comment (cyrichardson) [reply style-guide/blog-guidelines.rst]: fixed

File: style-guide/blog-guidelines.rst (hunk 4 above)
Comment (cyrichardson) [reply style-guide/blog-guidelines.rst]: fixed

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/build-b6539ca5d9/docs/style-guide/). :bow:

File: style-guide/blog-guidelines.rst [hunk 5]
```diff
@@ -0,0 +1,181 @@
...
+-  Avoid jargon! Rackers (and the entire tech industry) tend to speak in
+   jargon. Unless your post is aimed at a technical audience, use plain
+   English. There is never a reason to use business jargon.
+
+-  Link link link: A first product mention? Link to the product page. Outside
+   accolades? Link to it. Citing a study or source or article? Link to it.
+   Have we written about it before? Link to it. Mention a partner or customer?
+   Link to their site.
+
+-  Keep it concise: The Internet have shrunken everyone’s attention span.
```
Comment (stephamon) [inline style-guide/blog-guidelines.rst]: Sorry, should have been more clear in the last comment, "have" should also be "has"

Comment (cyrichardson) [reply style-guide/blog-guidelines.rst]: Add here to that image should contribute to content of the blog post.

File: style-guide/blog-guidelines.rst [hunk 6]
```diff
@@ -0,0 +1,181 @@
...
+
+- What is the goal of this developer blog? Describe it in two sentences or
+  less.
+
+  - Is it to be helpful and answer a question or offer a How-To?
+  - Is it to provide “thought leadership” or show our expertise in an area?
+    If so, describe what you’d like to convey.
+  - Is it something else altogether? If so, what? It’s possible that your
+    idea might be better presented as an article in the How-To or as a
+    technical or marketing white paper.
```
Comment (stephamon) [inline style-guide/blog-guidelines.rst]: Ooh we should probably nix the white paper suggestion, given that we don't have them anymore.

Comment (cyrichardson) [reply style-guide/blog-guidelines.rst]: fixed

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/build-1454639574/docs/style-guide/). :bow:

File: style-guide/blog-guidelines.rst (hunk 5 above)
Comment (cyrichardson) [reply style-guide/blog-guidelines.rst]: fixed

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/build-e87cb5a503/docs/style-guide/). :bow:

File: style-guide/blog-guidelines.rst (hunk 1 above)
Comment (ReneeRendon) [reply style-guide/blog-guidelines.rst]: Do we even recommend a word processor like Microsoft?

File: style-guide/blog-guidelines.rst (hunk 6 above)
Comment (ReneeRendon) [reply style-guide/blog-guidelines.rst]: " ... that idea might be better presented as an article on the How-To support site ..."

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/build-4001cd7dda/docs/style-guide/).

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/build-ff262a42ad/docs/style-guide/)

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/build-f3e158500c/docs/style-guide/)

File: style-guide/blog-guidelines.rst (hunk 6 above)
Comment (cyrichardson) [reply style-guide/blog-guidelines.rst]: fixed

File: style-guide/blog-guidelines.rst (hunk 1 above)
Comment (cyrichardson) [reply style-guide/blog-guidelines.rst]: fixed

File: style-guide/blog-guidelines.rst [hunk 7]
```diff
@@ -0,0 +1,198 @@
...
+
+- Images:
+
+  - Any image included in the blog post should contribute to the content of
+    that specific blog.
+  - Any image pulled from another source should provide a reference to the
+    source directly underneath the image.
+  - Add social media icons with their links for twitter, Facebook, and
+    LinkedIn at the bottom of the blog post. (Information Development provides
+    a template for this that you can add to your blog.)
```
Comment (catlook) [inline style-guide/blog-guidelines.rst]: - Images should directly support or illustrate blog content.  For example, a screen shot, diagram or table that illustrates the concept or technique being described.

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/build-bdb931a442/docs/style-guide/).

**Comment by cyrichardson** [comment]
With reviews from Stephanie, Cat, and Renee, I'm going to merge this one.

**Pull request merged**